src/VCAuthn/ACAPy/Models.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace VCAuthn.ACAPy
+{
+    public class CreatePresentationResponse
+    {
+        [JsonProperty("presentation_request")]
+        public JObject PresentationRequest { get; set; }
+            
+        [JsonProperty("created_at")]
+        public DateTime CreatedAt { get; set; }
+            
+        [JsonProperty("updated_at")]
+        public DateTime UpdatedAt { get; set; }
+            
+        [JsonProperty("thread_id")]
+        public string ThreadId { get; set; }
+            
+        [JsonProperty("presentation_exchange_id")]
+        public string PresentationExchangeId { get; set; }
+            
+        [JsonProperty("connection_id")]
+        public string ConnectionId { get; set; }
+            
+        [JsonProperty("initiator")]
+        public string Initiator { get; set; }
+            
+        [JsonProperty("state")]
+        public string State { get; set; }
+    }
+
+    public class WalletDidPublicResponse
+    {
+        public class ResponseResult
```

Comment:
Shall we take this class definition outside of WalletDidPublicResponse?

Reply:
Infact shall we reduce nesting around this object? And just have a single object as the result rather than a nested result object?

Reply:
I'd like to reduce nesting, but that's the ACAPy response model for this endpoint:
```
{
  "result": {
     "did": ...
  }
}```
It seems intuitive to keep all the parts of the response together, as they aren't used separately.

Thanks for reminding about this endpoint, as I need to add `~services` part to the presentation message with the contents of this response.